packages/cli/src/common/base-command.ts

```diff
@@ -1,14 +1,17 @@
 import Command from '@oclif/command'
 import { checkCurrentDirBoosterVersion } from '../services/project-checker'
+import { appendOnErrorsFile } from '../services/logger'
 
 export default abstract class BaseCommand extends Command {
   async init() {
-    await checkCurrentDirBoosterVersion(this.config.userAgent)
+    await checkCurrentDirBoosterVersion(this.config.version)
   }
 
-  async catch(err: any) {
-    // add any custom logic to handle errors from the command
-    // or simply return the parent class error handling
-    return super.catch(err)
+  async catch(fullError: any) {
+    const errorMessage = fullError.message.split("\n")[0].replace('Error:','')
+    const logRefMessage = "\n(You can see the full error logs in ./errors.log)"
+    const errorForFile = `\nboost ${this.id} ${this.argv.join(' ')}\n${fullError.message}`
+    appendOnErrorsFile(errorForFile)
```

Comment:
Should this be optional? I believe it would could get uncomfortable if the file is generated by default

Reply:
I was thinking of adding the flag `--verbose`, making the errors be displayed on the terminal in place of the file. What do you think @NickSeagull ?

Reply:
I think that is much better instead of a file, if the user wants a file they can always pipe the output

Reply:
Agree here. the `verbose` flag seems a good option